### DISCUSSION TITLE
Agent not pulling images during match

### DISCUSSION THREAD

**Discussion by CyberPickles** [SlickRick1987/AdultDVDEmpire.bundle#4] [General]
The content matching is working great & posters download correctly, but I can't get screenshots or gallery images to download at all.

Do you have any suggestions?

**Comment by SlickRick1987** [comment]
I have not coded for anything beyond the poster.  Can you tell me a little more about what you are looking for?  For instance, where are these stored in your set-up, do you save these in the Background?  Was the legacy ADE agent pulling those or do you have an example of an agent that is doing what you are hoping to find here?

**Reply by CyberPickles** [reply]
Sorry, there are options in the plex agent menu that asked to select how many screenshots and other images would download upon matching content. 

This may be a holdover in plex from me using another much older version of a similar agent that i have been using previously that no longer works. 

I'll have a look into my setup and see where my error is.

**Reply by SlickRick1987** [reply]
I have seen those options in the preferences file, but they were never important to me.  I will look at the legacy code and see if I can figure out what it was supposed to do.  If the pictures are supposed to come in as background art, it should code just like the posters.

**Comment by SlickRick1987** [comment]
The May 7th update should fix the issues.  I made sure to bring back all the functionality from Preferences.  So the GOOD_SCORE should be working now, the code will randomly pull your desired number of screencaps and/or gallery items into the Background if that is your preference, and the Studio As A Collection preference should be working now too.  Initial tests were working and no fatal errors so far, but if you experience issues, let me know.

**Reply by CyberPickles** [reply]
All working on my end.
I'm grateful for your work in getting these features implemented so quickly.
